app/routes/application.js

```diff
@@ -43,3 +44,18 @@ export default Ember.Route.extend({
     this.get('mainMap').resetBounds();
   },
 });
+
+Ember.Route.reopen({
```

Comment:
Oh, I see. That's code I copied from a tutorial. I thought it might be too cargo-cult. I think this only needs to happen in the index route. Please feel free to rewrite it.

Reply:
Thanks - just interested :)

Reply:
Honestly, I don't understand what reopen does. 😇